test/e2e/es_index_cleaner_test.go

```diff
@@ -71,19 +77,47 @@ func esIndexCleanerTest(t *testing.T, f *framework.Framework, testCtx *framework
 	}
 	defer portForw.Close()
 	defer close(closeChan)
-	err = SmokeTest("http://localhost:16686/api/traces", "http://localhost:14268/api/traces", "foo-bar", retryInterval, timeout)
+
+	esPod, err := GetPod("default", "elasticsearch", "elasticsearch", f.KubeClient)
 	if err != nil {
 		return err
 	}
-
-	err = WaitForCronJob(t, f.KubeClient, namespace, fmt.Sprintf("%s-es-index-cleaner", name), retryInterval, timeout)
```

Comment:
Currently removed the intermediate checks of whether the cronjob completed - as was focusing on the result. But wondering whether this should be added back in - just to check that the job also reports success.